README.markdown

```diff
@@ -88,8 +88,8 @@ TABLE: weak_guy;
 +---------------+-------------+------+-----+---------+----------------+
 | Field         | Type        | Null | Key | Default | Extra          |
 +---------------+-------------+------+-----+---------+----------------+
-| strong_guy_id | int(10)     | NO   | PRI | 0       |                |
-| weak_guy_id   | int(10)     | NO   | PRI | NULL    | auto_increment |
+| strong_guy_id | int(10)     | NO   |     | 0       |                |
```

Comment:
One of two has to be changed:
1. strong_guy_id is not PRI
2. weak_guy_id is not auto_increment

I just guessed it is 1., but change it to 2.